draft-ietf-quic-http.md

```diff
@@ -300,9 +301,9 @@ component of the URL.
 If a server presents a valid certificate and proof that it controls the
 corresponding private key, then a client will accept a secured TLS session with
 that server as being authoritative for all origins with the "https" scheme and a
-host identified in the certificate.  The host must be listed either as the CN
-field of the certificate subject or as a dNSName in the subjectAltName field of
-the certificate; see {{!RFC6125}}.  For a host that is an IP address, the client
```

Comment:
Can we just leave this text alone in the editorial PR, since there are already issue(s) tracking it?

Reply:
Okay, will attempt to revert.